### PR TITLE
BZ-1722775: Adding additional host names and ports for firewall config

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kalexand-rh Can you review this and make sure the wording is okay to match your updates for #17071? Thanks!

File: installing/installing_aws/installing-aws-customizations.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ in the AWS documentation. You can supply the keys when you run the installation
 program.
 ====
 * If you use a firewall, you must
-xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to access Red Hat Insights].
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow access to the sites] required for your {product-title} cluster.
```
Comment (kalexand-rh) [inline installing/installing_aws/installing-aws-customizations.adoc]: s/required for your {product-title} cluster./that your cluster requires access to.

**Review by kalexand-rh — COMMENTED** [review]
I'd modify a phrase, but this change matches the end result of the other change. Thanks!

**Comment by bergerhoffer** [comment]
I'm going to go ahead and merge this, since this wording is still accurate even without the updates in #17071.

**Pull request merged**